data/sender.go

```diff
@@ -127,13 +129,18 @@ func (s *Sender) send() {
 		r := s.lastStats.Report()
 		report := fmt.Sprintf("at %s: %s", pct.TimeString(r.Begin), FormatSentReport(r))
 		s.status.Update("data-sender-last", report)
-		s.logger.Info(report)
```

Comment:
That's where I find `Debug` log level useful – disabled by default, but may be priceless during debugging.